schedule/boot_to_snapshot.yaml

```diff
@@ -0,0 +1,13 @@
+name:           boot_to_snapshot
+description:    >
+    Test booting to automatically created snapshots.
+vars:
+    BOOT_HDD_IMAGE: 1
+    BOOT_TO_SNAPSHOT: 1
+    START_AFTER_TEST: create_hdd_textmode
+    # DESKTOP: textmode -> When added, an empty key appears first in the list
+    # HDD_1: %DISTRI%-%VERSION%-%ARCH%-%BUILD%-%DESKTOP%@%MACHINE%.qcow2 -> if added, openQA will not publish in UI
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

Reply:
I added it by analogy with the yast2_gui.yaml. But you're right, no need to have the commented lines here. Removed.